docs/CLI/index.md

```diff
@@ -10,45 +10,37 @@ Install `oras` using [Homebrew](https://brew.sh/):
 brew install oras
 ```
 
-## GoFish
-
-Install `oras` using [GoFish](https://gofi.sh/):
-
-```bash
-gofish install oras
-```
-
 ## Release artifacts
 
 Install from the latest [release artifacts](https://github.com/oras-project/oras/releases):
 
 ### Linux
 
 ```bash
-curl -LO https://github.com/oras-project/oras/releases/download/v0.12.0/oras_0.12.0_linux_amd64.tar.gz
+curl -LO https://github.com/oras-project/oras/releases/download/v0.13.0/oras_0.13.0_linux_amd64.tar.gz
```

Comment:
Is there interest to change these numbers in one place, and have them render here? E.g., it looks like you could use https://github.com/rosscdh/mkdocs-markdownextradata-plugin.

Reply:
Good idea. Just noticed this MKdocs plugin. It will be more efficient to maintain these common-used variables in multiple places. 
cc @shizhMSFT @jdolitsky @sajayantony @SteveLasker

Reply:
@vsoch IMO, we can start using this plugin from the next release. Shall we merge this first for a quick update to 0.13.0? I think we should also refactor the website and content from 0,14,0.

Reply:
That works for me! And if you want some help to try out the plug-in and make these changes I’d be happy to.